docker-compose.yml

```diff
@@ -18,9 +18,9 @@ services:
         environment:
             POSTGRES_USER: ${POSTGRES_USER}
             POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
-            POSTGRES_DB: ${POSTGRES_TEST_DB}
+            POSTGRES_DB: ${POSTGRES_DB}
         ports:
-            - ${POSTGRES_TEST_PORT}:5433
+            - ${POSTGRES_TEST_PORT}:5432
```

Comment:
Should we really default to the "standard" port? Wouldn't we run the risk of modifying our local DB unexpectedly? Like yeah I guess it doesn't really matter if it's local.

Reply:
The order here is:
`<port on your machine>`:`<port on the docker machine>`
The database inside the postgres docker container always runs on 5432, and we want to map that port to the one on the running machine. The way it was before meant we were mapping port 5433 inside the container to something on our machine, on which obviously nothing is running. The 5433 was actually a mistake that I left in back in the Go days and copy pasted when I moved the docker stuff over LOL

Reply:
_Oh yeah_ that's the actual syntax. Totally forgot. As you can probably tell, I was thinking the syntax is `<port>:<default>`.